### DISCUSSION TITLE
(dev) Preferred docstring style?

### DISCUSSION THREAD

**Discussion by weedySeaDragon** [freelawproject/bigcases2#278] [Q&A]
Is there a preferred style for the docstrings?  I see a mix in the code.
1. Google
2. ReStructured text
3. NumPy
4. EpyText

**Comment by mlissner** [comment]
It'd be nice to have a preference, but I don't have strong opinions and we don't use them for anything other than reading the code. My big thing is that they should be good, however you do them, and I'd rather not see another new style beyond what we already have. If somebody has strong opinions and wants to express them, I'd be happy to have that be the standard here and everywhere. (though it'd be a big lift to migrate existing code).
